noxfile.py

```diff
@@ -173,7 +173,7 @@ def install_bezier(session, debug=False, env=None):
 @nox.parametrize("check", [True, False])
 def update_generated(session, check):
     # Install all dependencies.
-    session.install(DEPS["Cython"])
+    session.install(DEPS["Cython"], DEPS["numpy"])
```

Comment:
This is the key fix! I wasn't installing **ANY** version of `numpy` in this virtualenv